### PR TITLE
sl-better-search

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
closes https://github.com/dart-lang/dartdoc/issues/837

**Comment by sethladd** [comment]
cc @devoncarew @keertip for review

File: lib/markdown_processor.dart [hunk 1]
```diff
@@ -82,6 +82,9 @@ class Documentation {
     });
     _asHtml = _asHtmlDocument.body.innerHtml;
 
+    // Fixes issue with line ending differences between mac and windows, affecting tests
+    if (_asHtml != null) _asHtml = _asHtml.trim();
```
Comment (devoncarew) [inline lib/markdown_processor.dart]: `_asHtml?.trim();`

:)

Comment (devoncarew) [inline lib/markdown_processor.dart]: Or some more appropriate use of a null aware operator...

**Comment by devoncarew** [comment]
lgtm

File: lib/markdown_processor.dart (hunk 1 above)
Comment (sethladd) [inline lib/markdown_processor.dart]: Ah, of course! Next time, for sure.

**Pull request closed without merging**